makeWidthDatacards.py

```diff
@@ -45,6 +45,8 @@ def parseOptions():
                       help='mLow: Low m4l boundary (def=220)')
    parser.add_option('--mHigh', type='float', dest='mHigh', default="1600",
                       help='mHigh: High m4l boundary (def=1600)')
+   parser.add_option('--sw', action='store_true', dest='scale_width', default=False,
```

Comment:
Shouldn't the default be True? Also, I'm not sure why we should ever ask to do the integral without "width". Doesn't it come with the risk of taking the wrong normalization? (I might have missed some thread here though)

Reply:
It is just to sync on the on-shell part of Anomalous Couplings with Heshy. However I agree that the default should be set to True.